druid/examples/custom_widget.rs

```diff
@@ -60,17 +60,22 @@ impl Widget<String> for CustomWidget {
     // Basically, anything that changes the appearance of a widget causes a paint.
     fn paint(&mut self, paint_ctx: &mut PaintCtx, data: &String, _env: &Env) {
         // Let's draw a picture with Piet!
-        // Clear the whole context with the color of your choice
-        paint_ctx.clear(Color::WHITE);
 
-        // Create an arbitrary bezier path
+        // Clear the whole widget with the color of your choice
         // (paint_ctx.size() returns the size of the layout rect we're painting in)
         let size = paint_ctx.size();
+        let rect = Rect::from_origin_size(Point::ORIGIN, size);
```

Comment:
I guess https://github.com/linebender/kurbo/pull/86 hasn't been released yet?

Reply:
It is not, will have to wait for the next piet release.

Reply:
ah that will be great to have